automatic/freecad/update_helper.ps1

```diff
@@ -23,13 +24,16 @@ param(
       $veri = ((($url64 -split('\/'))[-1]) -replace( "(x\d{2})|(_\d{2}\-py\d{2})|(\-)?([A-z])+?(\-)|(\.$ext)", ''))
       "veri -$veri-" | Write-Warning
       $DevRevision,$year,$month,$day = (($veri -replace('\-','.') ) -split('\.'))
-      [version]$DevVersion = (( Get-Content "$ScriptLocation\freecad.json" | ConvertFrom-Json ).dev) -replace("-$kind",'')
-      # Catch to make sure version will be the latest version
-      if (( $DevVersion.Minor -lt ($year.Substring(0,2)) ) -and ( $DevVersion.Build -gt $filler )) { [version]$DevVersion = "0.20.${filler}" }
-      if (($DevRevision -match "\d{5}") -and ($DevRevision -le "29192")){
-        "Revision is less than it was on date 6/20/2022 which could mean a new version minor" | Write-Warning
-        "Going to increase the version minor by 1" | Write-Warning
-        [version]$version = ( ( ($DevVersion.Major),(($DevVersion.Minor + 1)),($DevVersion.Build),($DevRevision) ) -join "." )
+      [version]$DevVersion = (( Get-Content "$ScriptLocation\freecad.json" | ConvertFrom-Json ).stable)
```

Comment:
isn't it better to call this variable StableVersion since it's taking the value from stable?

Reply:
also, is freecad.json auto updated? if so, how? otherwise it should be updated too since the current in the repo is outdated.

Reply:
The json file is updated by AU each time the script is run. This is a little fail-safe for if the stable version updates. The dev version will be auto updated to one version higher. It isn't called stable due to it being updated later in the script also to keep it simple.

Reply:
pardon if it's just due to my ignorance but I don't see DevVersion being assigned any new value, just read.
Also, shouldn't `$freecadMinor = ($DevVersion.Minor + 1)` happen when DevVersion >= freecadMinor? from what I understand it would only run when DevVersion > freecadMinor, so if nobody updates this script when 0.21 stable is out freecadMinor would still be 0.21 when it should be 0.22 (although next version might actually be 1.0 so all this would become invalid)

Reply:
This would all be simpler if FreeCAD left the current version.minor in the filename for the development build.

This is all done to makeup the version based on the current listing in chocolatey. I posted about asking if I could just change to using the date versioning, but no response. This PR was made based on that discussion and reply or lack of reply.